content/docs/localization/joining-the-localization-team.md

```diff
@@ -1,46 +1,115 @@
+* You must own an account on [scratch.mit.edu](https://scratch.mit.edu).
 
+{{< admonition info >}}
+**These languages do NOT accept new translators:** Italian, Japanese, Portuguese, Slovenian, Spanish.
+<!-- This list of languages is also found below. Remember to update both. -->
 {{< /admonition >}}
 
-**Thanks for being interested on volunteering to translate Scratch Addons to other languages!**
+If you're not a native speaker but consider yourself fluent enough, you may also request to join.  
+We only support languages that Scratch itself supports. We also fully support RTL languages, such as Arabic and Hebrew.
 
-Scratch Addons is a non-profit open source project where volunteer web developers maintain the extension and create its addons. 
+## Content available for translation
 
-Even if you haven't ever coded a single line of JavaScript, you can still help us achieve our goal of making everyone's experience with Scratch better!
+#### Scratch Addons browser extension
 
-**All you need to do is know both English and another language, and spend a few minutes a week translating any new sentences Scratch Addons added during that period.** 
+- **General Translation.** These are ~150 text strings which are used by the extension. Users can only see these translations for their language if 90% of general strings were translated.
 
-This way, anyone that speaks your language can use Scratch Addons in their native language, instead of having to understand English.
+- **Addons Translation.** These text strings include addon names, descriptions, settings, and other text strings used by addons. Users can only see these translations if the general translation is already available and 80% of addon strings were translated. There are over 1000 addon strings available for translation.
 
-**If you're interested in helping, please follow these steps:**
+![Screenshot of the Scratch Addons extension in Japanese](/assets/img/docs/transifex-general-vs-addons.png)
 
-- Go to [our Transifex page](https://app.transifex.com/scratch-addons/scratch-addons-extension/), and click on the blue "Help Translate Scratch Addons" button.
-- Use the same username as your Scratch account. Enter your email, a new password, then Sign Up.
-- Enter your username as your name and surname - **do not use your real name**.
-_Note: choose any option for "department" and "job title", we recommend "localization" and "individual contributor"_
-- On then next step, select "join an existing project".
-- Choose the language you want to help with.
+#### Scratch Addons website
 
-You'll soon be accepted into the Transifex team and you'll get email notifications when there are new strings to translate.
+Translators also have the possibility to translate the contents of the ScratchAddons.com website. For more information about translating the website, see [Translating](https://github.com/ScratchAddons/website-v2/wiki/Translating).

```

Comment:
ScratchAddons.com website is redundant. Best to put it as Scratch Addons website (or the website).

Reply:
Note that some readers might be visiting the ScratchAddons.com for the first time as they read.